Specs/Scene/MetadataTableSpec.js

```diff
@@ -0,0 +1,2291 @@
+import { defined } from "../../Source/Cesium.js";
+import { defaultValue } from "../../Source/Cesium.js";
+import { FeatureDetection } from "../../Source/Cesium.js";
+import { MetadataClass } from "../../Source/Cesium.js";
+import { MetadataEnum } from "../../Source/Cesium.js";
+import { MetadataTable } from "../../Source/Cesium.js";
+import { MetadataType } from "../../Source/Cesium.js";
+
+describe("Scene/MetadataTable", function () {
+  if (
+    !FeatureDetection.supportsBigInt64Array() ||
+    !FeatureDetection.supportsBigUint64Array() ||
+    !FeatureDetection.supportsBigInt() ||
+    typeof TextEncoder === "undefined"
+  ) {
+    return;
+  }
+
+  function createBuffer(values, type) {
```

Comment:
Consider factoring out these functions into a MetadataTester class so we could reuse it in future branches (e.g. implicit tiling + metadata)